### PR TITLE
feat(agent-control): L2 identity creation

### PR DESCRIPTION
**Pull request by rubenruizdegauna**
Create L2 identities using L1 identity token in Agent Control installation.

### CODE REVIEW

File: recipes/newrelic/infrastructure/super-agent/debian.yml [hunk 1]
```diff
@@ -519,58 +519,143 @@ install:
...
+                    -o "$TEMPORAL_FOLDER/response.json" \
+                    --data-binary @- \
+                    "$IDENTITY_CREATION_ENDPOINT"
+                )
+  
+                if [ $HTTP_CODE -eq 200 ]; then
+                  break
+                fi
+  
+                echo "Error creating the new system identity. The API endpoint returned $HTTP_CODE. Retrying ($RETRY/3)..."
```
Comment (sigilioso) [inline recipes/newrelic/infrastructure/super-agent/debian.yml:595]: Would it make sense to use slightly different error messages in order to help troubleshooting during the transition? For instance we could add L2 `Error creating L2 system identity...`

**Review by sigilioso — COMMENTED** [review]
LGTM. I've just left a question

File: recipes/newrelic/infrastructure/super-agent/debian.yml (hunk 1 above)
Comment (rubenruizdegauna) [reply recipes/newrelic/infrastructure/super-agent/debian.yml]: Good point! Changed in f6777411fa6b142a97ef758cb7ba86e18bf63739

**Review by sigilioso — APPROVED** [review]
LGTM

**Pull request merged**